addons/website_crm_partner_assign/models/res_partner.py

```diff
@@ -34,6 +34,7 @@ class ResPartnerActivation(models.Model):
 
     sequence = fields.Integer('Sequence')
     name = fields.Char('Name', required=True)
+    active = fields.Boolean(default=True)
```

Comment:
Could you increase module minor version ? As we change models, better indicate it in minor version in manifest :) . Don't forget to update upgrade accordingly.